ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js

```diff
@@ -0,0 +1,135 @@
+import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import {ContextualMenuDecorator} from '../ContextualMenuDecorator';
+import Button from '../../Button';
+
+const ContextualButton = ContextualMenuDecorator(Button);
+const items = new Array(3).fill().map((i, index) => `Option ${index + 1}`);
+
+describe('ContextualMenuDecorator Specs', () => {
+	test('should render component into FloatingLayer if open', () => {
+		const Root = FloatingLayerDecorator('div');
```

Comment:
How about moving this variable declaration above `describe`? This way you don't have to declare it at the beginning of each test.